testing/src/main/java/io/grpc/testing/TestUtils.java

```diff
@@ -180,6 +180,17 @@ public static File loadCert(String name) throws IOException {
   }
 
   /**
+   * Returns the given certificate resource in src/main/resources/certs/ as an {@code InputStream}.
+   *
+   * @param name name of a file in src/main/resources/certs/, e.g., {@code "ca.key"}.
+   *
+   * @since 1.8.0
+   */
+  public static InputStream loadCertFromResource(String name) throws IOException {
```

Comment:
I'd like to move this out of `TestUtils`, because `TestUtils` is a dumping ground. I was sort of thinking we could create a new class `TlsTesting` or similar for it: `TlsTesting.loadCert`.